msu/systems/mod_settings/settings_element.nut

```diff
@@ -0,0 +1,109 @@
+::MSU.Class.SettingsElement <- class
+{
+	Name = null;
+	ID = null;
+	static Type = "Element";
+	Flags = null;
+	Description = null;
+	Parent = null;
```

Comment:
Parent should be renamed to Page or at least to something that immediately tells what the type of this variable is. Of course the associated function should then be renamed to setPage etc.

Reply:
Sure